src/PHPCodeBrowser/CLIController.php

```diff
@@ -49,7 +49,7 @@
 
 namespace PHPCodeBrowser;
 
-use File_Iterator_Factory;
+use SebastianBergmann\FileIterator\Factory;
```

Comment:
I'd add `as File_Iterator_Factory` or similar, otherwise it's just way to generic.